miniapps/solvers/div_free_solver.cpp

```diff
@@ -0,0 +1,604 @@
+// Copyright (c) 2010-2020, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
+#include "div_free_solver.hpp"
+
+using namespace std;
+using namespace mfem;
+using namespace blocksolvers;
+using mfem::internal::mfem_hypre_ParCSRMatrixDropSmallEntries;
```

Comment:
Functions that are used by examples/miniapps should not be in the `mfem::internal` namespace. I think it is best to make this function a method in the `HypreParMatrix` class.